src/containers/Event/Settings/Settings.js

```diff
@@ -41,60 +43,74 @@ function Settings() {
-      </MuiPickersUtilsProvider>
-
-      <MuiPickersUtilsProvider utils={DateFnsUtils}>
-        <DateTimePicker
-          label="Event End Date"
-          inputVariant="outlined"
-          value={endDate}
-          onChange={setEndDate}
+        >
+          Event settings:
+        </Typography>
+        <TextField
+          label="Jitsi server"
+          name="jitsiServer"
+          fullWidth
+          value={jitsiServer}
+          onChange={(event) => setJitsiServer(event.target.value)}
+          variant="outlined"
           style={{ marginBottom: "20px" }}
         />
-      </MuiPickersUtilsProvider>
-      <FormControlLabel
-        control={
-          <Checkbox
-            checked={hasFreeMovement}
-            onChange={(event) => setHasFreeMovement(event.target.checked)}
-            name="freeMovement"
-            color="primary"
+        <MuiPickersUtilsProvider utils={DateFnsUtils}>
+          <DateTimePicker
+            label="Event Start Date"
+            inputVariant="outlined"
+            value={startDate}
+            onChange={setStartDate}
+            style={{ marginBottom: "20px" }}
           />
-        }
-        label="Enable users free movement between rooms"
-      />
-      <Button
-        variant="contained"
-        color="secondary"
-        size="large"
-        style={{ margin: "0 auto", fontWeight: 600 }}
-        type="submit"
-        onClick={() => {
-          updatePublicPeriod({ startDate, endDate });
-          setEventHasFreeMovement(hasFreeMovement);
-          setEventJitsiServer(jitsiServer);
-        }}
-      >
-        {`Update settings`}
-      </Button>
+        </MuiPickersUtilsProvider>
+
+        <MuiPickersUtilsProvider utils={DateFnsUtils}>
+          <DateTimePicker
+            label="Event End Date"
+            inputVariant="outlined"
+            value={endDate}
+            onChange={setEndDate}
+            style={{ marginBottom: "20px" }}
+          />
+        </MuiPickersUtilsProvider>
+        <FormControlLabel
+          control={
+            <Checkbox
+              checked={hasFreeMovement}
+              onChange={(event) => setHasFreeMovement(event.target.checked)}
+              name="freeMovement"
+              color="primary"
+            />
+          }
+          label="Enable users free movement between rooms"
+        />
+        <Button
+          variant="contained"
+          color="secondary"
+          size="large"
+          style={{ margin: "0 auto", fontWeight: 600 }}
+          type="submit"
+          onClick={() => {
+            updatePublicPeriod({ startDate, endDate });
+            setEventHasFreeMovement(hasFreeMovement);
+            setEventJitsiServer(jitsiServer);
+          }}
+        >
+          {`Update settings`}
+        </Button>
+      </div>
+      <div className="Edit-container">
+        <AddDoc eventId={event.eventId} />
+      </div>
+      <div className="Edit-container">
+        <Docs />
+      </div>
```

Comment:
IMHO it's better to add and delete files from the Docs container not in settings because you the users will access all the docs related commands on the docs tab an you don't need to repeat the doc preview code